django/db/migrations/operations/models.py

```diff
@@ -277,70 +278,50 @@ def deconstruct(self):
             kwargs
         )
 
+    def _get_model_key(self, remote_model, app_label, model_name):
```

Comment:
I'll rename to `_get_model_tuple` but I'll keep the `remote_model` naming as it's not necessarily a model name -- it could include an app label.